packages/online-shell/src/discover/Discover.tsx

```diff
@@ -24,7 +28,18 @@ export const Discover: React.FunctionComponent = () => {
     useDisplayItems()
 
   if (isLoading) {
-    return <HawtioLoadingCard />
+    return (
+      <PageSection variant={PageSectionVariants.light}>
+        <Panel variant='raised' className='discover-loading'>
+          <PanelHeader>Waiting for Hawtio Containers ...</PanelHeader>
+          <PanelMain>
+            <PanelMainBody>
+              <HawtioLoadingCard />
```

Comment:
Inside the panel, the loading card has a border and it doesn't look nice a bit.

Reply:
Maybe we don't need the panel and header at all, and you can directly use `<HawtioLoadingCard>` for alignment with the other views.

Reply:
I can make the panel invisible but given the loading can take ~7 or 8 seconds, I felt that the text was essential to explain to the user what was loading exactly. We could place an optional text header in `HawtioLoadingCard` at a later point?

Anyway I'll make the panel invisible then do a screenshot and you can see what you think.

Reply:
![Screenshot_20231127_105004](https://github.com/hawtio/hawtio-online/assets/1634180/5a9f1e1d-ab0e-40e5-8adb-16937130e0ba)

Reply:
ok, lets go with it for now.